appcircle_ios_appcenter_distribute/1.0.1/component.yaml

```diff
@@ -0,0 +1,95 @@
+platform: iOS
+buildPlatform:
+displayName: "App Center iOS Distribute"
+description: "Distribute IPA and dSYM files to App Center."
+docsUrl: https://docs.appcircle.io/workflows/ios-specific-workflow-steps#app-center-ios-distribution
+requiredComponents: "appcircle_ios_build_sign"
+precedingComponents: 
+followingComponents: "appcircle_export_build_artifacts"
```

Comment:
> @boztopuz Since this review is very repetitive, I will not give the same feedback in the rest, but can you provide the checks for all of them?

I mentioned the above situation in the previous review, but since the `export_build_artifacts` for which I did not add a comment were not corrected, I am adding them again for each one.
=> There is no any dependencies with `appcircle_export_build_artifacts`